backend/fleet_management/management/commands/populate_database.py

```diff
@@ -1,80 +1,122 @@
 import random
 
 from django.core.management.base import BaseCommand
-from tqdm import tqdm
-
 from django.contrib.auth.models import Group
-from ...constants import Groups
-
+from django.db import transaction
+from tqdm import tqdm
 
+from fleet_management.constants import Groups
 from fleet_management.factories import (
     CarFactory,
     DriveFactory,
     ProjectFactory,
     UserFactory,
 )
-from ...models import Project, User
 
 
 class Command(BaseCommand):
     """Populate database with fake items."""
 
     help = "Populates database with fake items."
 
+    @transaction.atomic
     def handle(self, *args, **options):
         driver_group = Group.objects.get(name=Groups.Driver.name)
         passenger_group = Group.objects.get(name=Groups.Passenger.name)
 
-        self.stdout.write(self.style.SUCCESS("Creating 5 cars"))
+        def info(message):
+            self.stdout.write(self.style.SUCCESS(message))
+
+        cars = [
+            CarFactory.make(
+                plates="UA000000",
+                description="Default Car",
+                fuel_consumption=10.0,
+                country="UA",
+            ),
+        ]
+        info("Creating 5 random cars")
```

Comment:
Clean database after populating has:
- 56 cars
- 52 drives
- 6 projects
- 19 users

Can you check that? Message in terminal can be misleading.

Reply:
The number of cars was a bug which I fixed just now, the other numbers will stay as they are because of the default object from each model.